CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+# 3.1.0
+> Nov 15, 2017
+* :nut_and_bolt: **New** Add prettier-eslint 💅
+* :nut_and_bolt: **New** Support react 15.x || 16.x 🕺🏼
```

Comment:
I would mark this change as a :boom: (breaking) because we cant guarantee that it wont break for people using React v16 (or can we @thalseth)?

Reply:
I use the component with react 16.1.1 and it works as expected 🎆 
The devDependencies are updated and the storybook use react 16.1.1.

Reply:
Yup...read comment above